app/views/incomes/ConfirmSaveIncomeView.scala.html

```diff
@@ -0,0 +1,98 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import includes.ptaHeader
+@import uk.gov.hmrc.play.views.formatting.Money._
+@import uk.gov.hmrc.urls.Link
+@import uk.gov.hmrc.play.views.html.helpers._
+@import uk.gov.hmrc.play.views.helpers.MoneyPounds
+@import org.joda.time.DateTime
+@import uk.gov.hmrc.play.views.html.layouts.loginStatus
+@import uk.gov.hmrc.tai.util.FormHelper
+@import uk.gov.hmrc.tai.util.TaxYearRangeUtil
+
+@import uk.gov.hmrc.tai.util.MonetaryUtil
+@import uk.gov.hmrc.tai.viewModels.GoogleAnalyticsSettings
+
+@this(mainTemplate: MainTemplate, form: FormWithCSRF)
```

Comment:
@Sachin300793 I can't diff this file as its been treated as a new file, can we load it up manually and ensure it looks the same as in Staging?

Reply:
Its name has been changed from confirm_save_Income.scala.html to ConfirmSaveIncomeView.scala.html . We removed partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever and changed main to MainTemplate